src/transformers/optimization.py

```diff
@@ -44,7 +49,15 @@ def get_constant_schedule(optimizer: Optimizer, last_epoch: int = -1):
     Return:
         `torch.optim.lr_scheduler.LambdaLR` with the appropriate schedule.
     """
-    return LambdaLR(optimizer, lambda _: 1, last_epoch=last_epoch)
+
+    lr_lambda = partial(_get_constant_schedule_lr_lambda)
```

Comment:
Why use a partial here, there is no need?

Reply:
Yea visual consistency with the others was the only reason, but that isn't much of one will remove